### PR TITLE
Try1

### PR DESCRIPTION
**Pull request by MaksimusCRS**
- [DEMO LINK](https://MaksimusCRS.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://MaksimusCRS.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by luckdenny — CHANGES_REQUESTED** [review]
- Reuse `stars` block from [Stars task](https://github.com/mate-academy/layout_stars)
and keep it operational. If you change `stars--4` modifier to `stars--2` or

**Review by luckdenny — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <div class="container__foto foto"></div>
+      <div class="container__text text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+      <div class="container__code code">Код товара: 195434</div>
+      <div class="container__rate rate">
```
Comment (ArtemTopolskyi) [inline src/index.html:19]: 5 пункт чеклиста

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+    <title>Product cards</title>
+    <link rel="stylesheet" href="Styles/reset.css">
+    <link rel="stylesheet" href="Styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <div class="container__foto foto"></div>
```
Comment (ArtemTopolskyi) [inline src/index.html:16]: используй img

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rate__review review">Отзывов: 5</div>
+      </div>
+      <div class="container__price price">
+        <div class="price__text">Цена:</div>
+        <div class="price__value">69 999 грн</div>
+      </div>
+      <button class="container__buy buy" data-qa="hover">КУПИТЬ</button>
```
Comment (ArtemTopolskyi) [inline src/index.html:33]:
```suggestion
      <button class="container__buy buy" data-qa="hover">
        Купить
      </button>
```
используй text-transform: uppercase

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

**Review by luckdenny — APPROVED** [review]

**Review by luckdenny — APPROVED** [review]

**Review by luckdenny — APPROVED** [review]

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,52 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img
+        src="/images/imac.jpeg"
+        class="container__foto"
+        alt="foto"
```
Comment (ArtemTopolskyi) [inline src/index.html:19]:
```suggestion
        alt="IMac photo"
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,52 @@
...
+      <div class="container__price">
+        <div class="container__letters">
+          Цена:
+        </div>
+        <div class="container__value">
+          69 999 грн
+        </div>
+      </div>
+      <button class="container__buy" data-qa="hover">
+        купить
```
Comment (ArtemTopolskyi) [inline src/index.html:48]:
```suggestion
        Купить
```

**Review by ArtemTopolskyi — APPROVED** [review]

**Pull request closed without merging**